Core/DISET/private/Transports/SSL/SocketInfo.py

```diff
@@ -4,202 +4,218 @@
 import time
 import copy
 import os.path
+import os
```

Comment:
no need to import `os.path` if you import `os` (but then careful in the rest of rest of the module)